README.md

```diff
@@ -320,6 +327,66 @@ Chewy.settings = {
   User.plugin :chewy_observe           # just for User
   ```
 
+#### Example of data request
+
+You can use the next query:
+
+```ruby
+def search
+  @users = UsersIndex.query(query_string: { fields: [:first_name, :last_name, :email, ...], query: search_params[:query], default_operator: 'and' })
+  render json: @users.to_json, status: :ok
+end
+```
+
+And add new user via Rails console:
+
+```ruby
+User.create(
+  first_name: "test1",
+  last_name: "test1",
+  email: 'test1@example.com',
+  # other fields
+)
+```
+
+It creates User index too:
+
+```ruby
```

Comment:
This doesn't seem like a Ruby snippet to me, but rather like the output from the index creation.